common/locales/en/settings.json

```diff
@@ -108,10 +108,12 @@
     "emailNotifications": "Email Notifications",
     "wonChallenge": "You won a Challenge!",
     "newPM": "Received Private Message",
+    "newPMInfo": "New Message from <%= name %>: <%= message %>",
     "sentGems": "Sent gems!",
     "giftedGems": "Gifted Gems",
-    "giftedGemsInfo": "<%= amount %> Gems - by <%= name %>",
+    "giftedGemsInfo": "<%= name %> gifted you <%= amount %> Gems",
     "giftedSubscription": "Gifted Subscription",
+    "giftedSubscriptionInfo": "<%= name %> gifted you a <%= months %> Subscription",
```

Comment:
The strings are the ones used for the content of the push notifications. Separating those into a separate PR doesn't seem to make much sense, since the one PR wouldn't work without the other.